### ISSUE TITLE
请求能在2*5090 (32G)上成功运行的示例配置参数

### ISSUE THREAD

**Issue by HaoliangZhou** [hiyouga/EasyR1#573]
**在使用最新的Docker环境 hiyouga/verl:ngc-th2.8.0-cu12.9-vllm0.11.0 运行 bash examples/qwen3_vl_2b_geo3k_grpo.sh 时，运行几个步骤后出现OOM（内存溢出）问题。

**是否有人能提供一个在相同GPU配置下成功运行的示例配置参数？

**当前sh文件:
python3 -m verl.trainer.main \
    config=examples/config.yaml \
    data.train_files=hiyouga/geometry3k@train \
    data.val_files=hiyouga/geometry3k@test \
    worker.actor.model.model_path=${MODEL_PATH} \
    trainer.experiment_name=qwen3_vl_2b_geo_grpo \
    trainer.logger=['console','swanlab'] \
    trainer.n_gpus_per_node=2 \
    worker.actor.fsdp.torch_dtype=bf16 \
    worker.actor.optim.strategy=adamw_bf16 \
    data.max_prompt_length=512 \
    data.rollout_batch_size=128 \
    worker.actor.global_batch_size=16 \
    worker.actor.micro_batch_size_per_device_for_update=1 \
    worker.actor.micro_batch_size_per_device_for_experience=1 \
    worker.rollout.gpu_memory_utilization=0.5 \
    worker.actor.offload.offload_params=False \
    worker.actor.offload.offload_optimizer=False 

**完整experiment_config.json如下
{
  "data": {
    "train_files": "hiyouga/geometry3k@train",
    "val_files": "hiyouga/geometry3k@test",
    "prompt_key": "problem",
    "answer_key": "answer",
    "image_key": "images",
    "video_key": "videos",
    "image_dir": null,
    "video_fps": 2.0,
    "max_prompt_length": 2048,
    "max_response_length": 2048,
    "rollout_batch_size": 512,
    "mini_rollout_batch_size": null,
    "val_batch_size": 1024,
    "format_prompt": "/workspace/project/EasyR1/examples/format_prompt/math.jinja",
    "override_chat_template": null,
    "shuffle": true,
    "seed": 1,
    "min_pixels": 262144,
    "max_pixels": 4194304,
    "filter_overlong_prompts": true,
    "filter_overlong_prompts_workers": 16
  },
  "worker": {
    "hybrid_engine": true,
    "actor": {
      "strategy": "fsdp",
      "global_batch_size": 128,
      "micro_batch_size_per_device_for_update": 1,
      "micro_batch_size_per_device_for_experience": 2,
      "max_grad_norm": 1.0,
      "clip_ratio_low": 0.2,
      "clip_ratio_high": 0.3,
      "clip_ratio_dual": 3.0,
      "loss_avg_mode": "token",
      "loss_type": "default",
      "ppo_epochs": 1,
      "padding_free": true,
      "dynamic_batching": true,
      "ulysses_size": 1,
      "use_torch_compile": true,
      "model": {
        "model_path": "/workspace/data/0_ckpt/LlamaFactory/Qwen3-VL-2B-Instruct",
        "tokenizer_path": "/workspace/data/0_ckpt/LlamaFactory/Qwen3-VL-2B-Instruct",
        "override_config": {},
        "enable_gradient_checkpointing": true,
        "trust_remote_code": false,
        "freeze_vision_tower": false
      },
      "optim": {
        "lr": 1e-06,
        "betas": [
          0.9,
          0.999
        ],
        "weight_decay": 0.01,
        "strategy": "adamw_bf16",
        "lr_warmup_ratio": 0.0,
        "lr_warmup_steps": null,
        "min_lr_ratio": null,
        "lr_scheduler_type": "constant",
        "training_steps": 60
      },
      "fsdp": {
        "enable_full_shard": true,
        "enable_cpu_offload": false,
        "enable_rank0_init": true,
        "use_orig_params": false,
        "torch_dtype": "bf16",
        "fsdp_size": -1,
        "mp_param_dtype": "bf16",
        "mp_reduce_dtype": "fp32",
        "mp_buffer_dtype": "fp32"
      },
      "offload": {
        "offload_params": true,
        "offload_optimizer": true
      },
      "global_batch_size_per_device": -1,
      "disable_kl": false,
      "use_kl_loss": true,
      "kl_penalty": "low_var_kl",
      "kl_coef": 0.01
    },
    "critic": {
      "strategy": "fsdp",
      "global_batch_size": 256,
      "micro_batch_size_per_device_for_update": 4,
      "micro_batch_size_per_device_for_experience": 16,
      "max_grad_norm": 1.0,
      "cliprange_value": 0.5,
      "loss_avg_mode": "token",
      "ppo_epochs": 1,
      "padding_free": false,
      "dynamic_batching": true,
      "ulysses_size": 1,
      "model": {
        "model_path": null,
        "tokenizer_path": null,
        "override_config": {},
        "enable_gradient_checkpointing": true,
        "trust_remote_code": true,
        "freeze_vision_tower": false
      },
      "optim": {
        "lr": 1e-06,
        "betas": [
          0.9,
          0.999
        ],
        "weight_decay": 0.01,
        "strategy": "adamw",
        "lr_warmup_ratio": 0.0,
        "lr_warmup_steps": null,
        "min_lr_ratio": null,
        "lr_scheduler_type": "constant",
        "training_steps": 60
      },
      "fsdp": {
        "enable_full_shard": true,
        "enable_cpu_offload": false,
        "enable_rank0_init": true,
        "use_orig_params": false,
        "torch_dtype": null,
        "fsdp_size": -1,
        "mp_param_dtype": "bf16",
        "mp_reduce_dtype": "fp32",
        "mp_buffer_dtype": "fp32"
      },
      "offload": {
        "offload_params": false,
        "offload_optimizer": false
      },
      "global_batch_size_per_device": -1
    },
    "ref": {
      "strategy": "fsdp",
      "fsdp": {
        "enable_full_shard": true,
        "enable_cpu_offload": true,
        "enable_rank0_init": true,
        "use_orig_params": false,
        "torch_dtype": null,
        "fsdp_size": -1,
        "mp_param_dtype": "bf16",
        "mp_reduce_dtype": "fp32",
        "mp_buffer_dtype": "fp32"
      },
      "offload": {
        "offload_params": false,
        "offload_optimizer": false
      },
      "micro_batch_size_per_device_for_experience": 2,
      "padding_free": true,
      "dynamic_batching": true,
      "ulysses_size": 1,
      "use_torch_compile": true
    },
    "reward": {
      "reward_function": "/workspace/project/EasyR1/examples/reward_function/math.py",
      "reward_function_kwargs": {},
      "skip_special_tokens": true,
      "num_cpus": 1,
      "reward_function_name": "compute_score"
    },
    "rollout": {
      "name": "vllm",
      "n": 5,
      "temperature": 1.0,
      "top_p": 1.0,
      "top_k": -1,
      "seed": 1,
      "limit_images": 0,
      "dtype": "bf16",
      "gpu_memory_utilization": 0.6,
      "ignore_eos": false,
      "enforce_eager": false,
      "enable_chunked_prefill": false,
      "tensor_parallel_size": 2,
      "max_model_len": null,
      "max_num_batched_tokens": 8192,
      "disable_log_stats": true,
      "disable_tqdm": false,
      "val_override_config": {
        "temperature": 0.6,
        "top_p": 0.95,
        "n": 1
      },
      "prompt_length": 2048,
      "response_length": 2048,
      "trust_remote_code": false
    }
  },
  "algorithm": {
    "gamma": 1.0,
    "lam": 1.0,
    "adv_estimator": "grpo",
    "disable_kl": false,
    "use_kl_loss": true,
    "kl_penalty": "low_var_kl",
    "kl_coef": 0.01,
    "kl_type": "fixed",
    "kl_horizon": 10000.0,
    "kl_target": 0.1,
    "online_filtering": false,
    "filter_key": "overall",
    "filter_low": 0.01,
    "filter_high": 0.99
  },
  "trainer": {
    "total_epochs": 15,
    "max_steps": null,
    "project_name": "easy_r1",
    "experiment_name": "qwen3_vl_2b_geo_grpo",
    "logger": [
      "file",
      "wandb"
    ],
    "nnodes": 1,
    "n_gpus_per_node": 2,
    "max_try_make_batch": 20,
    "critic_warmup": 0,
    "val_freq": 5,
    "val_before_train": true,
    "val_only": false,
    "val_generations_to_log": 3,
    "save_freq": 5,
    "save_limit": 3,
    "save_model_only": false,
    "save_checkpoint_path": "/workspace/project/EasyR1/checkpoints/easy_r1/qwen3_vl_2b_geo_grpo",
    "load_checkpoint_path": null,
    "ray_timeline": null,
    "find_last_checkpoint": true
  }
}
